docs/OV_Runtime_UG/ShapeInference.md

```diff
@@ -1,4 +1,6 @@
-# Changing input shapes {#openvino_docs_OV_UG_ShapeInference}
+# Changing Input Shapes {#openvino_docs_OV_UG_ShapeInference}
+
+This guide presents information about operating on Input Shapes, divided into C++ and Python sections.
```

Comment:
```suggestion
```
Checked the paragraph in the original Introduction section - it can be a good summary of this article, so I'm removing this intro sentence here. :)